add/add.js

```diff
@@ -23,6 +24,11 @@ let commands = {
 }
 
 let add = () => {
+  if (!fs.existsSync('package.json')) {
```

Comment:
@alocke12992 now checking if the project has a package.json before adding any tools lol

Reply:
Smart move